pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/BrokersImpl.java

```diff
@@ -366,8 +368,31 @@ public void healthcheck() throws PulsarAdminException {
     }
 
     @Override
+    @Deprecated
     public CompletableFuture<Void> healthcheckAsync() {
+        return healthcheckAsync("V1");
+    }
+
+    @Override
+    public void healthcheck(String topicVersion) throws PulsarAdminException {
```

Comment:
keep input type`TopicVersion` instead `String`

Reply:
sure where would be put the TopicVersion class that client, broker and tools package can share it.